command/server.go

```diff
@@ -944,6 +955,8 @@ func (c *ServerCommand) Run(args []string) int {
 	}
 
 	// Initialize the Service Discovery, if there is one
```

Comment:
This comment should be beneath the WaitGroup line.

Reply:
Implemented in https://github.com/hashicorp/vault/pull/8123/commits/2b0f648391e270e6c60ecfdbcd67f76c75170133